taichi_3d_gaussian_splatting/GaussianPointCloudRasterisation.py

```diff
@@ -370,6 +365,17 @@ def gaussian_point_rasterisation(
         pixel_offset_in_tile = pixel_offset - tile_id * 256
         pixel_u = tile_u * 16 + pixel_offset_in_tile % 16
         pixel_v = tile_v * 16 + pixel_offset_in_tile // 16
+
+        camera_intrinsics_mat = ti.Matrix(
+            [[camera_intrinsics[row, col] for col in ti.static(range(3))] for row in ti.static(range(3))])
+        # K: [fx, 0, cx; 0, fy, cy; 0, 0, 1]
+        # inv_K: [1/fx, 0, -cx/fx; 0, 1/fy, -cy/fy; 0, 0, 1]
+        inv_camera_intrinsics_mat = ti.math.mat3([
```

Comment:
See comment in the backward func.

Reply:
Overall lgtm. Some small comments are left.
Looking forward to see the results after updating the place where `gaussian_point_rasterisation` and `gaussian_point_rasterisation_backward` are called.